gradle.properties

```diff
@@ -19,3 +19,6 @@ POM_DEVELOPER_NAME2=Paweł Urban
 
 android.useAndroidX=true
 android.enableJetifier=false
+
+artifactoryUsername=android-gradle
+artifactoryPassword=TQy+dwFh6vdHt
```

Comment:
Should these be env variables from 1Password?

Reply:
They should be in 1password, aren't they?

Reply:
I meant should the script itself fetch it from the developers/ci machines env var rather than have it plaintext in the repo

Reply:
This is not running on any ci, only local machine as it is not updated very often. We should however do that for the android repo, not just for the artifactory credentials but others as well.

Reply:
ok, removed from repo and added in 1Password